src/expression.rs

```diff
@@ -98,13 +99,44 @@ impl Index<usize> for SetDirective {
     }
 }
 
+/// A ConvDirective is a command of the form "to hex|dec|bin|oct|dec".
+#[derive(Debug, PartialEq, Eq)]
+pub struct ConvDirective {
+    expr: Expr,
+    radix: String
+}
+
+impl ConvDirective {
+    pub fn value(&self, ans: i64) -> Result<i64, CorkError> {
+        eval::eval_expr(&self.expr, ans)
+    }
+
+    pub fn radix(&self) -> Result<FormatRadix, CorkError> {
```

Comment:
This function will become trivial if the type of `radix` is `FormatRadix`. Also the `Result` will be un-neccessary.

Reply:
Ok.